pom.xml

```diff
@@ -1,70 +1,229 @@
+                            </dependency>
+                            <dependency>
+                                <groupId>org.junit.jupiter</groupId>
+                                <artifactId>junit-jupiter-api</artifactId>
+                                <version>[5.2.0]</version>
+                            </dependency>
+
+
+                            <dependency>
+                                <groupId>ch.ethz.rtc.kernel</groupId>
+                                <artifactId>rtc</artifactId>
+                                <scope>system</scope>
+                                <systemPath>${project.basedir}/lib/rtc.jar</systemPath>
+                                <version>1.2.beta.100</version>
+                            </dependency>
+
+                        </dependencies>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>2.12.4</version>
+                        <configuration>
+                            <skipTests>true</skipTests>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>mpa</id>
+            <dependencies>
+                <dependency>
+                    <groupId>ch.ethz.rtc.kernel</groupId>
+                    <artifactId>rtc</artifactId>
+                    <scope>system</scope>
+                    <systemPath>${project.basedir}/lib/rtc.jar</systemPath>
+                    <version>1.2.beta.100</version>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>build-helper-maven-plugin</artifactId>
+                        <version>1.7</version>
+                        <executions>
+                            <execution>
+                                <id>add-source</id>
+                                <phase>generate-sources</phase>
+                                <goals>
+                                    <goal>add-source</goal>
+                                </goals>
+                                <configuration>
+                                    <sources>
+                                        <source>src/mpa_ext</source>
+                                    </sources>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>exp</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
```

Comment:
I do not think you have to repeat it so many times. Just add the execution to the executions of the first one.

Reply:
I think I don't get your point here. What do you mean?

Reply:
<executions>
 <execution>
  1
</execution>
<execution>
  2
</execution>

</executions>

Reply:
I am not familiar with these concepts, please help me out.
As far as I understand, these executions are specific to the profile they are nested in. Having all executions listed together with the first one means moving them into the tests profile, right? In this case, how to call the correct execution from the map profile?